src/web/docusaurus/docusaurus.config.js

```diff
@@ -120,7 +120,7 @@ const config = async () => {
               items: [
                 {
                   label: 'Documentation',
-                  href: 'https://docusaurus.io/docs/',
+                  href: 'https://telescope.cdot.systems/docs/overview/',
```

Comment:
We have 2 environments for this 
One staging: https://dev.telescope.cdot.systems/docs/overview/
One production: https://telescope.cdot.systems/docs/overview/

I don't think it's a good idea to redirect always to production.

I believe you can do relatives path [see](https://docusaurus.io/docs/api/themes/configuration#footer-links)

Reply:
Hi @Kevan-Y ,

Thank you for your feedback, I changed the link, hopefully it's okay :)

Reply:
@mnosov622 it looks like you've switched to using an absolute URL to staging now, which also won't work.  What @Kevan-Y is saying is that we need this to work no matter where we host this site.  The link needs to take into account the fact that we host this same site on different URLs.

Reply:
Hi @humphd ,

Oh, yes, I'm sorry, i get it. Please let me know if everything is fine now :)